src/main/java/reposense/RepoSense.java

```diff
@@ -34,12 +34,15 @@
     private static final int SERVER_PORT_NUMBER = 9000;
     private static final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("E MMM d HH:mm:ss yyyy z");
     private static final String VERSION_UNSPECIFIED = "unspecified";
+    private static final String MESSAGE_ELAPSED_TIME = "Elapsed processing time: %f seconds";
```

Comment:
Instead of seconds, I prefer a `**hour**minute**second` way, which can be more intuitive.

Reply:
ok noted.